pkg/lighthouse/ensure.go

```diff
@@ -35,7 +35,7 @@ func Ensure(status reporter.Interface, kubeClient kubernetes.Interface, dynClien
 		status.Success("Created lighthouse service account and role")
 	}
 
-	if created, err := scc.Ensure(dynClient, operatorNamespace); err != nil {
+	if created, err := ocp.EnsureRBAC(dynClient, kubeClient, operatorNamespace); err != nil {
 		return err // nolint:wrapcheck // No need to wrap here
 	} else if created {
 		status.Success("Updated the privileged SCC")
```

Comment:
Should this change (elsewhere as well)?

```suggestion
		status.Success("Updated the privileged OCP roles")
```